src/util/misc.js

```diff
@@ -1215,5 +1215,111 @@
       }
       return new fabric.Group([a], { clipPath: b, inverted: inverted });
     },
+
+    /**
+     * @memberOf fabric.util
+     * @param {Object} prevStyle first style to compare
+     * @param {Object} thisStyle second style to compare
+     * @param {boolean} includeLineStyles whether to check overline, underline, and line-through properties
+     */
+    hasStyleChanged: function(prevStyle, thisStyle, includeLineStyles) {
```

Comment:
let's call `includeLineStyles` something like forSvg or forTextSpans.
lineStyles is specific and does not cover what the difference was there for.